Gemfile

```diff
@@ -21,7 +21,7 @@ manageiq_plugin "manageiq-schema"
 
 # Unmodified gems
 gem "activerecord-session_store",       "~>2.0"
-gem "activerecord-virtual_attributes",  "~>7.1.0"
+gem "activerecord-virtual_attributes",  "~>7.1.1"
```

Comment:
the goal is to release 7.2, since that will be rails 7.2 friendly. That should fix up a lot of stuff.

We were thinking we may want `~>7.2.0.0` to give us the flexibility to upgrade to newer features. This may be overkill and we can discuss.